pyproject.toml

```diff
@@ -77,30 +77,30 @@ dependencies = [
 
 [project.optional-dependencies]
 dev = [
-    "black",
-    "build",
-    "bumpver",
-    "djlint",
-    "isort",
-    "pre-commit",
-    "pyjwt",
+    "black==23.3.0",
```

Comment:
Sorry, pinning all versions in the `dev` extra is not what I meant by #2182.
The problem is that this is not sufficient since a lot of sub-dependencies are still not pinned, plus it makes hard to add or remove specific dev dependencies.
Could you leave all lined unchanged except `pytest-testmon`?

Reply:
Ah, sure